### DISCUSSION TITLE
Where in build.sbt to put PlayKeys.devSettings?

### DISCUSSION THREAD

**Discussion by Rapierian** [playframework/playframework#11462] [Q&A] [answered]
Hello, [per this page](https://www.playframework.com/documentation/2.8.x/AkkaHttpServer) the solution to my `play.core.server.ServerStartException: No ServerProvider configured with key 'play.server.provider'` Error would seem to be to add a `PlayKeys.devSettings += "play.server.provider" -> "play.core.server.AkkaHttpServerProvider"` somewhere in my build.sbt file, but I can't seem to find a location to do so that doesn't generate a "Reference to undefined setting"

**Comment by mkurz** [comment, marked as answer]
Can you post (at least part) of you `build.sbt`? Did you enable the `PlayScala` or the `PlayJavaPlugin`?

**Comment by Rapierian** [comment]
Ah, that was it. I didn't realize that the project didn't have the PlayScala plugin enabled.
